package.json

```diff
@@ -19,7 +19,10 @@
   },
   "pre-commit": "pre-commit",
   "jest": {
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "transform": {
+      "^.+\\.jsx?$": "babel-jest"
```

Comment:
AFAIK it will automagically included if present. Am I wrong?

Reply:
![screen shot 2017-08-25 at 20 14 31](https://user-images.githubusercontent.com/22868432/29715566-10de3370-89d2-11e7-811a-6b7eb4dff4e5.png)

oopss.. yeah you're right 😢  thanks for letting me know!